### PR TITLE
feat: add http polling method

### PR DESCRIPTION
**Pull request by leanmendoza**
# What?
Enable support to using the client part by http-polling and not requiring `Socket.IO`.
- Add POST `/http-polling/request` and GET `/http-polling/:requestId` endpoitns

If request is created by http-polling, it doesn't have a `socketId`. In that case it holds the response until the client poll it

### CODE REVIEW

**Comment by cazala** [comment]
i'm not very familiar with the auth server, but if it is being run as multiple instances behind a load balancer, storing the request in memory is not going to work, no? This would be OK only for a single instance of the service.

**Comment by kuruk-mm** [comment]
> i'm not very familiar with the auth server, but if it is being run as multiple instances behind a load balancer, storing the request in memory is not going to work, no? This would be OK only for a single instance of the service.
> 
> 
> 
> EDIT: im talking this line https://github.com/decentraland/auth-server/pull/65/files#diff-8560c285fba362f05604f1a231f4e68c9d236de17be776381e214ef82c310ac2R233

Using Socket.IO should not have the same problem? You need to be in the same instance between the FrontApp - Server - ExplorerApp.

LoadBalancing of the service should require adding shared memory between those if Im not wrong.

File: test/integration/with-polling.spec.ts [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+  await new Promise(resolve => {
+    const handleOnConnect = () => {
+      resolve(undefined)
+    }
+
+    authDappSocket.on('connect', handleOnConnect)
+  })
+}
+
+test('when sending a request message with an invalid schema (http-polling-version)', args => {
```
Comment (fzavalia) [inline test/integration/with-polling.spec.ts:70]: Writing `(http-polling-version)` is redundant as the test runner will output the name of the file to identity the tests.

```
 PASS  test/integration/with-polling.spec.ts (37.869 s)
```

**Review by fzavalia — CHANGES_REQUESTED** [review]
Added some comments to address some simple things but overall looks good to me. As an extra thing, you should add this new http endpoints to the README as well as how they should be used.

**Review by fzavalia — APPROVED** [review]

**Pull request merged**